src/test/java/fake/FkProducer.java

```diff
@@ -20,7 +20,7 @@
  * SOFTWARE.
  */
 
-package org.eocqrs.kafka.fake;
+package fake;
```

Comment:
The idea was to build fake producers, consumers to use them as [Fake objects](https://www.yegor256.com/2014/09/23/built-in-fake-objects.html) in tests, in order to don't mock. Not only in our project, but in projects where we will add eo-kafka dependency.

Like jcabi-github does: https://github.com/jcabi/jcabi-github/tree/master/src/main/java/com/jcabi/github/mock